kotlin/src/main/kotlin/com/linecorp/armeria/internal/common/kotlin/ArmeriaCoroutineUtil.kt

```diff
@@ -67,3 +66,16 @@ internal fun callKotlinSuspendingMethod(
 
     return future
 }
+
+/**
+ * Converts [Flow] into [Publisher].
+ * @see FlowCollectingPublisher
+ */
+internal fun <T : Any> Flow<T>.asPublisher(
+    executorService: ExecutorService,
+    ctx: ServiceRequestContext
+): Publisher<T> = FlowCollectingPublisher(this, ctx, newCoroutineCtx(executorService, ctx))
```

Comment:
How about passing `executorService` to `FlowCollectingPublisher` and use the executor when subscribing to `DefaultStreamMessage`?

Reply:
`DefaultStreamMessage#subscribe()` only accepts `EventExecutor`.
Maybe we can modify `asPublisher` method to accept `EventExecutor` but I wanted it to accept `ctx.blockingTaskExecutor()` too so that it can support `@Blocking` annotation.

Unfortunately, however, `@Blocking` annotation support is not possible without a breaking change in `ResponseConverterFunctionProvider` like...:
```java
ResponseConverterFunction createResponseConverterFunction(Type responseType,
                                                          ResponseConverterFunction responseConverter,
                                                          boolean useBlockingExecutor); // newly added
```

How do you think? If you disagree, I will change the method to accept `EventExecutor` to pick up your suggestion 😄.

Reply:
I had a second thought and it seems better to do it when there be the needs! 😅
Or at least in a separate PR.

Changed the code as per your comment!